Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM docker.io/library/rockylinux:9@sha256:d7be1c094cc5845ee815d4632fe377514ee6ebcf8efaed6892889657e5ddaaa6
+LABEL maintainer="Red Hat"
+
+WORKDIR /src
+RUN dnf -y install \
```

Comment:
Where do we actually install `createrepo`? I'm little bit confused here

Reply:
This PR is only about switching the image, i.e. `createrepo_c` is still being installed via pip. The relevant change would come in the #544 where the lack of the package would actually become apparent due to multi-stage.
If needed I can drop the mention of the package as the reason from commit message https://github.com/containerbuildsystem/cachi2/pull/528/commits/a2b4db7778f0ef3d4a4d2b6ad734290ab06a7a0e

Reply:
ak, thanks

Reply:
just an FYI, I updated the commit message in question to drop any mention of the createrepo_c dependency as advertised in my previous comment.